rct229/rulesets/ashrae9012019/section16/section16rule1.py

```diff
@@ -0,0 +1,120 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.ruleset_model_factory import produce_ruleset_model_description
+from rct229.rulesets.ashrae9012019 import BASELINE_0
+from rct229.rulesets.ashrae9012019.data_fns.table_G3_9_1_fins import table_G3_9_1_lookup
+from rct229.rulesets.ashrae9012019.data_fns.table_G3_9_2_fins import table_G3_9_2_lookup
+from rct229.rulesets.ashrae9012019.data_fns.table_G3_9_3_fins import table_G3_9_3_lookup
+from rct229.schema.config import ureg
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.pint_utils import CalcQ
+from rct229.utils.std_comparisons import std_equal
+
+
+class Section16Rule1(RuleDefinitionListIndexedBase):
+    """Rule 1 of ASHRAE 90.1-2019 Appendix G Section 16 (Elevators)"""
+
+    def __init__(self):
+        super(Section16Rule1, self).__init__(
+            rmds_used=produce_ruleset_model_description(
+                USER=False,
+                BASELINE_0=True,
+                PROPOSED=True,
+            ),
+            each_rule=Section16Rule1.ElevatorRule(),
+            index_rmd=BASELINE_0,
+            id="16-1",
+            description="The elevator peak motor power shall be calculated according to the equation in Table G3.1-16.",
+            ruleset_section_title="Elevators",
+            standard_section="Section G3.1",
+            is_primary_rule=True,
+            rmd_context="ruleset_model_descriptions/0",
+            list_path="$.buildings[*].elevators[*]",
+        )
+
+    def is_applicable(self, context, data=None):
+        rmd_b = context.BASELINE_0
+        rmd_p = context.PROPOSED
+
+        elevators_list_b = find_all("$.buildings[*].elevators[*]", rmd_b)
+        elevators_list_p = find_all("$.buildings[*].elevators[*]", rmd_p)
+
+        return elevators_list_p and elevators_list_b
+
+    class ElevatorRule(RuleDefinitionBase):
+        def __init__(self):
+            super(Section16Rule1.ElevatorRule, self).__init__(
+                rmds_used=produce_ruleset_model_description(
+                    USER=False,
+                    BASELINE_0=True,
+                    PROPOSED=False,
+                ),
+            )
+
+        def get_calc_vals(self, context, data=None):
+            elevator_b = context.BASELINE_0
+
+            total_floors_served_b = getattr_(
+                elevator_b, "elevators", "number_of_floors_served"
+            )
```

Comment:
add `assert_` to make sure the `total_floors_served_b` is greater than 1. 
It is unclear how the schema will define this parameter - so far in the develop branch, it is described as a number - which is incorrect. We should update it to integer in the schema, and put `minimum` to 1

Reply:
Good point. Addressed.